src/OVAL/public/oval_results.h

```diff
@@ -724,6 +724,10 @@ void oval_result_criteria_node_iterator_free(struct oval_result_criteria_node_it
  * @name Evaluators
  * @{
  */
+/**
+ * @memberof oval_result_criteria_node
+ */
+oval_result_t oval_result_criteria_node_negate(struct oval_result_criteria_node *node, oval_result_t result);
```

Comment:
Please do not expose this to public. You only need this in `src/OVAL/oval_probe.c`. However, in another commit, you are making the `oval_probe.c` to depend on `results/oval_results_impl.h`. So, I think `oval_result_criteria_node_negate` should go to `oval_results_impl.h` as well.